specs/basic-card-payment.html

```diff
@@ -142,9 +142,14 @@
 
       <table>
         <tr><th>Identifier String</th><th>Description</th></tr>
-        <tr><td>visa</td><td>VISA cards</td></tr>
-        <tr><td>...</td><td>...</td></tr>
-      </table>
+        <tr><td>visa</td><td>Visa (Credit, Debit, Delta or Electron)</td></tr>
+        <tr><td>mastercard</td><td>MasterCard and EuroCard</td></tr>
+        <tr><td>amex</td><td>American Express</td></tr>
+        <tr><td>discover</td><td>Discover</td></tr>
+        <tr><td>maestro</td><td>Maestro</td></tr>
+	    <tr><td>diner</td><td>Diners Club</td></tr>
+        <tr><td>jcb</td><td>JCB</td></tr>
+	  </table>
     </section>
```

Comment:
I suggest we namespace these for this payment method because it's quite possible (likely) that we'll have a "tokenized card payments" or "encrypted card payments" spec pretty soon.

```html
<tr><td>basic-card-visa</td><td>Visa (Credit, Debit, Delta or Electron)</td></tr>
<tr><td>basic-card-mastercard</td><td>MasterCard and EuroCard</td></tr>
<tr><td>basic-card-amex</td><td>American Express</td></tr>
<tr><td>basic-card-discover</td><td>Discover</td></tr>
<tr><td>basic-card-maestro</td><td>Maestro</td></tr>
<tr><td>basic-card-diners</td><td>Diners Club</td></tr>
<tr><td>basic-card-jcb</td><td>JCB</td></tr>
```